app/controllers/pages_controller.rb

```diff
@@ -5,7 +5,12 @@ class PagesController < ApplicationController
   before_action :load_page
   layout "page_with_sidebar"
 
-  def index; end
+  def index
+    if !current_user && Page.default_page.present?
+      redirect_to page_path(Page.default_page.url_slug)
+      nil
```

Comment:
```suggestion
      redirect_to page_path(Page.default_page.url_slug) and return
```
is a more common pattern in rails

Reply:
I think this causes rubocop issue...? Will check later

Reply:
Nvm this `and return` way of returning does not violates rubocop.

But this does:
```
redirect_to page_path(Page.default_page.url_slug)
return
```